### DISCUSSION TITLE
Congratulations

### DISCUSSION THREAD

**Discussion by michel-briand** [nobiot/org-transclusion#235] [General]
The best org Emacs add-on I've seen in years ! [10 years or more ;)] Thank you !!!

**Comment by nobiot** [comment]
@michel-briand Thank you for your comment! May I ask how you use it? I would like to understand how people are really using it for inspiration:)

**Reply by michel-briand** [reply]
Hello,
I use org-mode with the source block feature to create graphviz diagrams for example.
I use org-transclusion to have my diagram source in another file.

`
#+transclude: [[s_ntp.dot]]  :src dot :rest ":file s_ntp.pdf :exports results"
`

**Reply by nobiot** [reply]
@michel-briand Thank you!
